tcms/report/views.py

```diff
@@ -416,9 +441,9 @@ def _report_data_context(self):
                                         self)._report_data_context()
             data.update(summary_header_data)
 
-            build_ids = [build['build'] for build in data['builds']]
+            build_ids = next(build['build'] for build in data['builds'])
```

Comment:
Can you explain this change ? I'm not sure I understand why we need it.

Reply:
In this case don't use the next() function but iterate over the values with a for loop and create the build_ids list.